baekjoon/피보나치-수-5.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+const input = fs.readFileSync('/dev/stdin').toString().split(' ');
+const n = Number(input);
+
+// let n = 10;
+
+function f(n) {
+  if (n === 0) return 0;
+  if (n === 1) return 1;
+  return f(n - 1) + f(n - 2);
+}
+console.log(f(n));
```

Comment:
```suggestion
function fibonacci(n) {
  if (n === 0) return 0;
  if (n === 1) return 1;
  return fibonacci(n - 1) + fibonacci(n - 2);
}
console.log(fibonacci(n));
```
다른 곳에서는 함수 명을 문제 주제에 맞게 잘 정해주셨는데 여기선 함수명을 명확하게 해주시지 않아서 조금 아쉽네요 그 외에는 문제 풀이도 깔끔하시고 다른 문제도 군더더기 없다고 생각됩니다 :)